tests/jballerina-unit-test/src/test/resources/test-src/annotations/deprecation_annotation.bal

```diff
@@ -394,3 +394,11 @@ public function testDeprecatedAnonStructAsStructField() {
     _ = employee5.address;  // warning
     _ = employee5.address.line02;   // warning
 }
+
+public function testDeprecatedAnonRecordFieldWithVarRef() {
```

Comment:
Is this correct? Testing spread also? Shall we just make this a standalone test that tests key-value pairs also?

Reply:
I thought of grouping record variable reference usages (including ref with spread). Do you mean to separate key-value, var-ref, and record var ref into separate functions?

Reply:
I thought VarRef meant variable name field here. 

What I meant anyway was having one test that tests passing fields in different ways: key-value, variable name field, and spread field.